_posts/2019-02-20-NUnit3TestAdapter-3.13.md

```diff
@@ -0,0 +1,6 @@
+NUnit3TestAdapter version 3.13.0 is released.
+
+This release focuses on enhancements: improving and fixing the generation of NUnit3 Test Results XML and removing internal properties.  It also ensures that the VSIX version works with VS2019. 
```

Comment:
> NUnit3 Test Results XML

I think elsewhere we call it 'Test Result XML' with no plural.

Since you mention the VSIX and 2019, do you think it's a good idea to repeat the bit in the release notes that it's officially deprecated by the VSTest team?